pt-BR/mail/chrome/messenger/addressbook/abMailListDialog.dtd

```diff
@@ -46,7 +46,7 @@
 <!ENTITY ListDescription.label          "Descrição:">
 <!-- See bug 58485, when we implement drag and drop, add 'or drag addresses' back in -->
 <!ENTITY AddressTitle.label             "Forneça os endereços para adicioná-los à lista de emails:">
-<!ENTITY RemoveButton.label             "Remove">
+<!ENTITY RemoveButton.label             "Remover">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY RemoveButton.label             "Excluir">